src/app/Interface/country.ts

```diff
@@ -0,0 +1,4 @@
+export interface CountryI {
```

Comment:
We already created a Product interface with a leading I instead of a trailing one. Can we stick to one common interface naming convention?

Reply:
changed naming to ICountry